configuration/items.md

```diff
@@ -35,7 +44,9 @@ Items are defined in the following syntax:
 itemtype itemname ["labeltext"] [<iconname>] [(group1, group2, ...)] [["tag1", "tag2", ...]] [{bindingconfig}]
 ```
```

Comment:
A list of parts and direct links to these (Type, Label, ...) should follow directly after this.